packages/bento-design-system/src/Form/Form.tsx

```diff
@@ -39,34 +41,37 @@ export function createForm(
     secondaryButton,
     error,
     actionsSize = config.defaultActionsSize,
+    autoFocus,
   }: Props) {
     return (
-      <Box as="form" onSubmit={(e) => e.preventDefault()}>
-        <ContentBlock maxWidth={700}>
-          <Stack space={config.formSpacing}>
-            {(title || description) && (
-              <Stack space={config.headerSpacing}>
-                {title &&
-                  (config.headerTitle.kind === "display" ? (
-                    <Display size={config.headerTitle.size}>{title}</Display>
-                  ) : (
-                    <Headline size={config.headerTitle.size}>{title}</Headline>
-                  ))}
-                {description && <Body size={config.headerDescriptionSize}>{description}</Body>}
-              </Stack>
-            )}
-            {children}
-            {(submitButton || secondaryButton) && (
-              <Actions
-                size={actionsSize}
-                primaryAction={submitButton}
-                secondaryAction={secondaryButton}
-                error={error}
-              />
-            )}
-          </Stack>
-        </ContentBlock>
-      </Box>
+      <FocusScope contain restoreFocus autoFocus={autoFocus}>
```

Comment:
```suggestion
      <FocusScope autoFocus={autoFocus}>
```

We only want `autoFocus` here, we don't need `contain` and `restoreFocus` since this is not a `Modal`